src/javaharness/java/arcs/builtinparticles/CaptureEntity.java

```diff
@@ -14,13 +13,13 @@
 public class CaptureEntity extends ParticleBase {
 
   private PortableJsonParser parser;
-  private final ClipboardSurface clipboardSurface;
+  private final ClipboardService clipboardService;
 
   public CaptureEntity(/*EntityObserver entityObserver*/ PortableJsonParser parser,
-      ClipboardSurface clipboardSurface) {
+      ClipboardService clipboardService) {
     //    entityObserver.registerListener(this);
     this.parser = parser;
-    this.clipboardSurface = clipboardSurface;
+    this.clipboardService = clipboardService;
```

Comment:
nit - could you, please, delete the entityObserver comments too? i don't think we need to keep it.